src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -33,28 +33,22 @@
  * Handles communication with the keyserver.
  */
 public class Rscc {
+  public static final int PACKAGE_SIZE = 10000; // needed, since a static method access it.
```

Comment:
@jpduloch I had to make this one public, since it is needed in ReliableSocketProfile.java. I was unable to access the property with normal getters.